platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```
@@ -199,7 +200,11 @@ public WebClient getWebClientForSubject(Subject subject) throws SecurityServiceE
     public T getClientForBasicAuth(String username, String password)
             throws SecurityServiceException {
         T client = getNewClient();
-        RestSecurity.setUserOnClient(username, password, WebClient.client(client));
+        try {
+            RestSecurity.setUserOnClient(username, password, WebClient.client(client));
+        } catch (UnsupportedEncodingException e) {
+            LOGGER.warn("Encoding error", e);
```

Shouldn't this be at least an error? Should we throw an exception?